src/App.js

```diff
@@ -1,24 +1,179 @@
 import './App.css';
+import TodoList from './TodoList';
+import { React, useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+const Box = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 60px;
+  padding-left: 40px;
+  padding-right: 40px;
+  border: 1px solid gray;
+  border-radius: 1rem;
+  width: 300px;
+  height: 500px;
+  background-color: #ffffff;
+  position: absolute;
+`;
+
+//제목, input 부분
+const TodoHeader = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+`;
+
+const InputForm = styled.div`
+  width: 380px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 5px;
+  .submit {
+    border: none;
+    background-color: transparent;
+    &:hover {
+      cursor: pointer;
+      transform: scale(1.7);
+    }
+  }
+`;
+
+const Title = styled.div`
+  border-top: 1px solid gray;
+  width: 380px;
+  height: 30px;
+  font-size: large;
+  font-weight: bold;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fefbf3;
+`;
+
+//Todo List 부분
+const TodoListContainer = styled.div`
+  border-top: 1px dashed gray;
+  width: 380px;
+  height: 150px;
+  overflow: auto;
+  display: flex;
+  justify-content: center;
+`;
+
+//Done List 부분
+const DoneListContainer = styled.div`
+  border-top: 1px dashed gray;
+  width: 380px;
+  height: 150px;
+  overflow: auto;
+  display: flex;
+  justify-content: center;
+`;
+
+const App = () => {
+  const [inputValue, setInputValue] = useState('');
+  const [todos, setTodos] = useState(
+    () => JSON.parse(window.localStorage.getItem('todos')) || []
+  );
+  const [dones, setDones] = useState(
+    () => JSON.parse(window.localStorage.getItem('dones')) || []
+  );
+  const [todoCnt, setTodoCnt] = useState(
+    () => JSON.parse(window.localStorage.getItem('count')) || 0
+  );
+
+  useEffect(() => {
+    window.localStorage.setItem('todos', JSON.stringify(todos));
+  }, [todos]);
+  useEffect(() => {
+    window.localStorage.setItem('dones', JSON.stringify(dones));
+  }, [dones]);
+  useEffect(() => {
+    window.localStorage.setItem('count', JSON.stringify(todoCnt));
+  }, [todoCnt]);
+
+  const onChange = (event) => {
+    const {
+      target: { value },
+    } = event;
+    setInputValue(value);
+  };
+
+  const onSubmit = (event) => {
+    event.preventDefault();
+    if (inputValue === '') {
```

Comment:
이부분을 
```suggestion
    if (!inputValue) {
```
이런식으로 바꾸는거도 괜찮을 것 같아요!